mozilla/addons-server
# set CinderDecision.action_date before sending emails

Fixes: mozilla/addons#15112



### Description

Fixes bug that lead to emails from Cinder not having appeal links/blub

### Context

Regression from https://github.com/mozilla/addons/issues/15074 - we don't set CinderDecision.action_date until after the emails have been sent (thinking: we'd want it only to be set when everything completed successfully); but CinderDecision.can_be_appealed is called when generating the email, and relies on action_date being set.

### Testing

Long way:
- Report some content from your local addons-server with Cinder integration set-up, that is handled in Cinder (e.g. extension for reason Other).
- Make a decision on that content in Cinder (e.g. addon disable)
- Copy the webhook response from Cinder admin and replay it locally
- Check FakeEmail in django admin and see there is an appeal link in the email sent to the content owner (e.g. add-on owner)

Short way:
- In django shell, find any `CinderDecision` instance and set it's `action_date` to `None`
- call `<instance>.process_action()`
- Check FakeEmail in django admin and see there is an appeal link in the email sent to the content owner (e.g. add-on owner)
- (the action_date should have been set to `datetime.now()` too)

### Checklist
